src/app/info/InfoPage.tsx

```diff
@@ -30,8 +30,8 @@ const InfoPage: React.FC<QueryType> = ({ data, error }) => {
         console.error(error);
```

Comment:
can we use the logger defined in `@logger/logger` to make sure we're sending the logs to cloudwatch?